### DISCUSSION TITLE
cannot mount usb stick on fat client

### DISCUSSION THREAD

**Discussion by justinb72** [ltsp/ltsp#184] [Q&A]
Hi all,

I setup ltsp on debian buster : everything is fine, except an error when I put an usb stick on fat client: cannot mount volume, not authorized to perform operation.

I found some things, including this without success :
https://superuser.com/questions/1151993/usb-automount-fail-not-authorized

My lts.conf :
[Default]
LDM_DIRECTX=True
LDM_PASSWORD_HASH=True
FSTAB_0="10.10.10.5:/home/PUBLIC /home/PUBLIC nfs defaults 0 0"
CUPS_SERVER=10.10.10.5
KEEP_SYSTEM_SERVICES=cups
LOCALDEV=True
LOCALDEV_DENY_USB=False
USE_LOCAL_SWAP=True
#Also try the following
#MODULE_01 = "usb-uhci"
#MODULE_02 = "usb-storage"
#MODULE_03 = "sd_mod"

Messages on fat client :
![usberror](https://user-images.githubusercontent.com/56481660/66703544-0aeddb80-ed14-11e9-9bf5-2a31a4d26e07.png)

Any idea ?
Thanks, Justin

**Comment by alkisg** [comment]
Hi Justin,

two **very** different versions of LTSP currently exist. The old one is called LTSP5 and is what Debian stable has. But it's minimally supported in the mailing lists, not here. If you want to use LTSP5, read the  [LTSP5 resources](https://ltsp.github.io#ltsp5-resources) section of https://ltsp.github.io/, and send a mail to the mailing lists there.

The new LTSP19+ version is the future; you can install it in Debian Buster using a PPA. If you want to use it, remove LTSP5 and install LTSP19+ using [the installation instructions](https://github.com/ltsp/ltsp/wiki/installation).

**Comment by justinb72** [comment]
Hi Alkis,

Indeed, much better :-)

Thanks !
